downloads/index.md

```diff
@@ -398,7 +397,7 @@ Checksums for this release are available in both, [SHA256](https://julialang-s3.
 
  We're currently testing release candidates for Julia v{{upcoming_release_short}}, an upcoming minor release in the 1.x series of releases. We encourage developers and interested users to try it out and report any issues they encounter. As a prerelease, it should not be considered production-ready; it's intended to give users a chance to try out {{upcoming_release_short}} with their code before the full release.
 
-Checksums for this release are available in both, [SHA256](https://julialang-s3.julialang.org/bin/checksums/julia-{{upcoming_release}}.sha256) and [MD5](https://julialang-s3.julialang.org/bin/checksums/julia-{{upcoming_release}}.md5) formats.
+[SHA256 checksums](https://julialang-s3.julialang.org/bin/checksums/julia-{{upcoming_release}}.sha256) | [MD5 checksums](https://julialang-s3.julialang.org/bin/checksums/julia-{{upcoming_release}}.md5)
```

Comment:
Do we not want to link NEWS.md here too? I actually feel like it's more useful in this case than for the others. Same question for the GitHub release, I suppose, though I'm not sure when exactly that would be useful in general.

Reply:
I had forgotten these are actual github tags. Yeah I'll add them